debian/control

```diff
@@ -29,19 +29,19 @@ Description: All packages for Dell Networking S6000-ON
 
 Package: opx-platform-config-mlnx-sn2410
 Architecture: any
-Replaces: opx-nas-acl
+Replaces: opx-nas-acl, logrotate, apt, rsyslog, dpkg, redis-server
```

Comment:
This replacing of `dpkg` files scared me so much that I looked into how others do it.

Configuration files are expected to belong to only one package. Please use [`dpkg-divert`](https://manpages.debian.org/stretch/dpkg/dpkg-divert.1.en.html) to override the config files instead of replacing them.

* [Documentation](https://manpages.debian.org/stretch/dpkg/dpkg-divert.1.en.html)
* [Here's a bug report to Linux Mint explaining why](https://bugs.launchpad.net/linuxmint/+bug/1317033)
* [Here's how to do it](https://serverfault.com/a/279356)
* [`preinst` example](https://github.com/grml/grml-etc-core/blob/master/debian/preinst#L45)
* [`postrm` example](https://github.com/grml/grml-etc-core/blob/master/debian/postrm#L18)